### PR TITLE
Update for Rescript with some new points

### PR DESCRIPTION
**Pull request by hugihlynsson**
We also need to change the name of this project ✏️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,45 @@
...
-    ~~let map = f =>
-      fun
-      | Some(v) => Some(f(v))
-      | None => None;~~
-    ```
-  This syntax is hard to parse and it is deprecated in latest versions of the language.
-12. When working with JSON create decoders and encoders with the `bs-json` lib
+# The Avo Rescript Style Guide
+
+At [Avo](https://github.com/avohq) we use Rescript as our primary programming language. To allow the development team to deliver better code and reviews, we've come up with a set of guidelines:
```
Comment (aleks-tpom6oh) [inline README.md]: Should we use avo.app link instead of github?

File: README.md [hunk 2]
```diff
@@ -1,54 +1,45 @@
...
+
+1. File names, just like modules, should be in PascalCase: `EventsList.res`
+1. Define meaningful types for every variant and polymorphic variant you introduce. For example, when defining `MergeBranch` you would create `type branchName = string` and the definition would look like: `MergeBranch(branchName)` instead of `MergeBranch(string)`
```
Comment (aleks-tpom6oh) [inline README.md:6]: Every point tuned out to be 1. 😄

Comment (hugihlynsson) [reply README.md:6]: Haha yeah. It's a funny feature of markdown—they'll end up correctly numbered and this allows you to more easily re-arrange them without having to re-number every item. What do you think?

File: README.md (hunk 1 above)
Comment (hugihlynsson) [reply README.md]: I'm up for that!
```suggestion
At [Avo](https://www.avo.app/) we use Rescript as our primary programming language. To allow the development team to deliver better code and reviews, we've come up with a set of guidelines:
```

File: README.md (hunk 2 above)
Comment (aleks-tpom6oh) [reply README.md:6]: Oh really? Confusing, but let's keep it then

**Review by aleks-tpom6oh — APPROVED** [review]
